packages/@okta/vuepress-site/guides/sign-into-spa/section11/index.md

```diff
@@ -1,8 +1,19 @@
 ---
 title: Use the Access Token
 ---
-## Use the Access Token
 
-After your user is authenticated, your application has an access token in local storage that was issued by your Okta authorization server. You can use this token to authenticate requests made for protected resources on your server. 
+Your app will need to send requests to one or more APIs to perform actions or retrieve information.
+
+After a user signs in, your application stores an access token issued by Okta. By attaching this token to outgoing requests, your API can authenticate them (ensure the user is signed in to perform an action) and authorize them (ensure the user is allowed to do an action).
+
+On your front-end (this single-page app), make sure you place the access token in the HTTP `Authorization` header of outgoing requests using this format:
```

Comment:
On your front-end (this single-page app), make sure that you place the access token in the HTTP `Authorization` header of outgoing requests using this format: